pkg/reconciler/wasmmodule/wasmmodule.go

```
@@ -119,8 +186,128 @@ func (r *Reconciler) createService(ctx context.Context, module *api.WasmModule)
+				// Secrets are mounted at a standard location
+				volumeMap[vol.Name] = fmt.Sprintf("/var/run/secrets/%s", vol.Name)
+			} else if vol.PersistentVolumeClaim != nil {
+				// PVCs are mounted at a standard location
+				volumeMap[vol.Name] = fmt.Sprintf("/var/run/volumes/%s", vol.Name)
+			}
+		}
+
+		for _, vm := range wm.Spec.VolumeMounts {
+			hostPath, ok := volumeMap[vm.Name]
+			if !ok {
+				return "", fmt.Errorf("volume mount %s references undefined volume %s", vm.MountPath, vm.Name)
```

golangci-lint (2.5.0)

[error] do not define dynamic errors, use wrapped static errors instead: "fmt.Errorf(\"volume mount %s references undefined volume %s\", vm.MountPath, vm.Name)"

(err113)